lib/presentation/ui/custom/text_input_form.dart

```diff
@@ -1,14 +1,21 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_base_rootstrap/int_constants.dart';
+import 'package:flutter_base_rootstrap/presentation/resources/locale/generated/l10n.dart';
+import 'package:flutter_base_rootstrap/string_constants.dart';
 import 'package:flutter_base_rootstrap/presentation/resources/resources.dart';
 
 class TextInputForm extends StatelessWidget {
   final String labelText;
   final String hintText;
+  final bool isPassword;
+  final FieldValidator? validator;
 
   const TextInputForm({
     Key? key,
     required this.labelText,
     required this.hintText,
+    this.isPassword = false,
```

Comment:
WDYT about using a different name for this, I get the idea but you don't "obscure" the text only if it's a password, maybe a credit card CCV can be "obscured", I was thinking more about `isSensitive` or `isSensitiveField`

Reply:
I understand what you're proposing, in this case, since there's no other sensitive field type, this was the selected name, but of course we can change it because it's also a good idea